kafka-iceberg/apache-flink/README.md

```diff
@@ -0,0 +1,327 @@
+# Streaming data from Kafka to Iceberg with Apache Flink
+
+`@rmoff 2024-07-02`
+
+_👉 See the supporting blog post at https://www.decodable.co/blog/_
+
+## Run it all
```

Comment:
Would be good to have a sentence here what "all" entails, i.e. what are the components of this example.